server/control.go

```diff
@@ -266,14 +266,20 @@ func (c Controller) runOnce(ctx context.Context, leaderKey string, tick <-chan t
 		return err
 	}
 
-	re, err := inf.Storage().GetRebootsEntries(ctx)
+	var reboot *cke.RebootQueueEntry
+	disabled, err := inf.Storage().IsRebootQueueDisabled(ctx)
 	if err != nil {
 		return err
 	}
-	metrics.UpdateReboot(len(re))
-	var reboot *cke.RebootQueueEntry
-	if len(re) > 0 {
-		reboot = re[0]
+	if !disabled {
+		re, err := inf.Storage().GetRebootsEntries(ctx)
+		if err != nil {
+			return err
+		}
+		metrics.UpdateReboot(len(re))
```

Comment:
When disabled, the metrics will not be updated.  Is this good behavior?

Reply:
sorry, fixed